.mergify.yml

```diff
@@ -12,4 +12,4 @@ pull_request_rules:
     actions:
       merge:
         method: squash
-        
+        strict: false
```

Comment:
Seems like `strict: true` is actually a good idea, especial with installer since it is pretty easy for two changes to be incompatible and get merged in. Prow already does `strict: true` basically on istio/istio. Although maybe mergify's version of it isn't as good